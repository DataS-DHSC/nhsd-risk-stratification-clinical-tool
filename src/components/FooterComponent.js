import React from 'react';
import { Footer } from 'nhsuk-react-components';
import {
  ROUTE_SUPPORT_FORM,
  ROUTE_COOKIES_POLICY,
  ROUTE_ACCESSIBILITY,
  ROUTE_TERMS_OF_USE,
  ROUTE_OPEN_SOURCE,
  ROUTE_SUPPORTED_BROWSERS,
  ROUTE_CE_MARK,
  ROUTE_CLINICAL_GUIDANCE,
  ROUTE_PATIENT_GUIDANCE,
} from '../constants/routes';

const FooterComponent = () => (
  <Footer visuallyHiddenText={false}>
    <Footer.List id="footer-list">
      <Footer.ListItem
        href={ROUTE_CLINICAL_GUIDANCE}
        target="docs"
        rel="noopener nofollow noreferrer"
        id="footer-clinical-guidance"
      >
        Clinical Guidance
      </Footer.ListItem>
      <Footer.ListItem
        href={ROUTE_PATIENT_GUIDANCE}
        target="docs"
        rel="noopener nofollow noreferrer"
        id="footer-guidance-for-patients"
      >
        Guidance for patients
      </Footer.ListItem>
      <Footer.ListItem
        href={ROUTE_COOKIES_POLICY}
        target="docs"
        rel="noopener nofollow noreferrer"
        id="footer-cookies-policy"
      >
        Cookies
      </Footer.ListItem>
      <Footer.ListItem
        href={ROUTE_SUPPORT_FORM}
        rel="noopener nofollow noreferrer"
        target="docs"
        id="footer-contact-us"
      >
        Contact us
      </Footer.ListItem>
      <Footer.ListItem
        href={ROUTE_ACCESSIBILITY}
        target="docs"
        rel="noopener nofollow noreferrer"
        id="footer-accessibility-statement"
      >
        Accessibility statement
      </Footer.ListItem>
      <Footer.ListItem
        href={ROUTE_TERMS_OF_USE}
        target="docs"
        rel="noopener nofollow noreferrer"
        id="footer-terms-of-use"
      >
        Terms of use
      </Footer.ListItem>
      <Footer.ListItem
        href={ROUTE_OPEN_SOURCE}
        target="docs"
        rel="noopener nofollow noreferrer"
        id="footer-open-source"
      >
        Open source
      </Footer.ListItem>
      <Footer.ListItem
        href={ROUTE_CE_MARK}
        target="docs"
        rel="noopener nofollow noreferrer"
        id="footer-ce-mark-label"
      >
        Legal information
      </Footer.ListItem>
      <Footer.ListItem
        href={ROUTE_SUPPORTED_BROWSERS}
        target="docs"
        rel="noopener nofollow noreferrer"
        id="footer-supported-browsers"
      >
        Supported browsers
      </Footer.ListItem>
    </Footer.List>
    <Footer.Copyright id="copyright-element">
      &copy; Crown copyright
    </Footer.Copyright>
  </Footer>
);

export default FooterComponent;
