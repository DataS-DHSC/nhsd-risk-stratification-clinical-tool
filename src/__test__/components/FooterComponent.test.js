import React from 'react';
import { render } from '@testing-library/react';
import FooterComponent from '../../components/FooterComponent';

const setup = () => render(<FooterComponent />);

describe('FooterComponent', () => {
  it('should match snapshot ', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
