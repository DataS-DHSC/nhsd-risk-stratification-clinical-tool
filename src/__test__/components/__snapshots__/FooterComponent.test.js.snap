// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent should match snapshot  1`] = `
<DocumentFragment>
  <footer>
    <div
      class="nhsuk-footer"
    >
      <div
        class="nhsuk-width-container"
      >
        <ul
          class="nhsuk-footer__list"
          id="footer-list"
        >
          <li
            class="nhsuk-footer__list-item"
          >
            <a
              class="nhsuk-footer__list-item-link"
              href="https://digital.nhs.uk/coronavirus/risk-assessment/clinical-tool/guidance-for-clinicians"
              id="footer-clinical-guidance"
              rel="noopener nofollow noreferrer"
              target="docs"
            >
              Clinical Guidance
            </a>
          </li>
          <li
            class="nhsuk-footer__list-item"
          >
            <a
              class="nhsuk-footer__list-item-link"
              href="https://digital.nhs.uk/coronavirus/risk-assessment/clinical-tool/information-for-patients"
              id="footer-guidance-for-patients"
              rel="noopener nofollow noreferrer"
              target="docs"
            >
              Guidance for patients
            </a>
          </li>
          <li
            class="nhsuk-footer__list-item"
          >
            <a
              class="nhsuk-footer__list-item-link"
              href="/page/cookies/policy"
              id="footer-cookies-policy"
              rel="noopener nofollow noreferrer"
              target="docs"
            >
              Cookies
            </a>
          </li>
          <li
            class="nhsuk-footer__list-item"
          >
            <a
              class="nhsuk-footer__list-item-link"
              href="https://www.nhs.uk/contact-us/covid-risk"
              id="footer-contact-us"
              rel="noopener nofollow noreferrer"
              target="docs"
            >
              Contact us
            </a>
          </li>
          <li
            class="nhsuk-footer__list-item"
          >
            <a
              class="nhsuk-footer__list-item-link"
              href="/page/accessibility"
              id="footer-accessibility-statement"
              rel="noopener nofollow noreferrer"
              target="docs"
            >
              Accessibility statement
            </a>
          </li>
          <li
            class="nhsuk-footer__list-item"
          >
            <a
              class="nhsuk-footer__list-item-link"
              href="/page/terms-of-use"
              id="footer-terms-of-use"
              rel="noopener nofollow noreferrer"
              target="docs"
            >
              Terms of use
            </a>
          </li>
          <li
            class="nhsuk-footer__list-item"
          >
            <a
              class="nhsuk-footer__list-item-link"
              href="/page/open-source"
              id="footer-open-source"
              rel="noopener nofollow noreferrer"
              target="docs"
            >
              Open source
            </a>
          </li>
          <li
            class="nhsuk-footer__list-item"
          >
            <a
              class="nhsuk-footer__list-item-link"
              href="/page/ce-mark-label"
              id="footer-ce-mark-label"
              rel="noopener nofollow noreferrer"
              target="docs"
            >
              Legal information
            </a>
          </li>
          <li
            class="nhsuk-footer__list-item"
          >
            <a
              class="nhsuk-footer__list-item-link"
              href="/page/supported-browsers"
              id="footer-supported-browsers"
              rel="noopener nofollow noreferrer"
              target="docs"
            >
              Supported browsers
            </a>
          </li>
        </ul>
        <p
          class="nhsuk-footer__copyright"
          id="copyright-element"
        >
          Â© Crown copyright
        </p>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
